platform.js

```diff
@@ -415,12 +416,12 @@ module.exports = {
         });
       }
       else {
-        var lnxCommand = "df --block-size=K --output=avail " + module.exports.getInstallDir();
+        var lnxCommand = "df -k " + dir + " | awk 'NR==2 {print $4}'";
 
```

Comment:
@tejohnso I tested the command on ubuntu and it returns what we need. I have also run RC cache that uses the function on ubuntu and it works fine. cheers